### DISCUSSION TITLE
Recover original study name to load .db file

### DISCUSSION THREAD

**Discussion by LewsTherin511** [optuna/optuna#3958] [Q&A] [answered]
Hello, I have kind of a noob question!
I created a study to optimize a model with Optuna, which produced a .db file with the same name as the study_name.  

The problem is that I'm trying to load the results by using:
```python
study = optuna.create_study(study_name=study_name,
                                storage=f"sqlite:///{results_folder}/results.db",
                                directions=["maximize", "maximize"],
                                load_if_exists=True)
```

but I renamed the original .db file, and I can't remember what was its original name (i.e., the original study_name value).

I seem to understand that I can rename the file and use the new file name in the "storage" argument when loading the study, but I should use the original value of study_name. In case I don't, I get a message saying that:

```
[W 2022-08-31 14:55:26,962] Study instance does not contain completed trials.
[W 2022-08-31 14:55:26,964] Your study does not have any completed trials.
```

Is there any way I can get it back from the .db file?
Thanks!

**Comment by nzw0301** [comment, marked as answer]
If `results.db` has only one study, we can see the original `study_name` by running the following command from the command line interface:

```bash
optuna studies --storage sqlite:///results.db
```

In the output, `name` column shows all `study_name`s.
```
+----------+---------------+----------+---------------------+
| name     | direction     | n_trials | datetime_start      |
+----------+---------------+----------+---------------------+
| my_study | ('MINIMIZE',) |       20 | 2022-09-02 18:38:09 |
+----------+---------------+----------+---------------------+
```

---

Ref: API docs of the command line interface: https://optuna.readthedocs.io/en/stable/reference/cli.html

**Reply by LewsTherin511** [reply]
Awesome, thank you very much!

**Reply by nzw0301** [reply]
I'm glad to hear that. Thank you for marking my reply as an answer!
